next-jdbc/src/ragtime/next_jdbc.clj

```diff
@@ -17,9 +17,14 @@
 
 ;; Returns unqualified maps as some databases (e.g. Oracle) can only return them
 (defn- get-table-metadata* [^Connection c]
-  (-> (.getMetaData c)
-      (.getTables (.getCatalog c) nil nil (into-array ["TABLE"]))
-      (rs/datafiable-result-set c {:builder-fn rs/as-unqualified-lower-maps})))
+  (let [metadata (.getMetaData c)
+         quote-str (.getIdentifierQuoteString metadata)]
+    {:tables
+     (-> metadata
+         (.getTables (.getCatalog c) nil nil (into-array ["TABLE"]))
+         (rs/datafiable-result-set c {:builder-fn rs/as-unqualified-lower-maps}))
+     :quote (when (not= quote-str " ")
+              quote-str)}))
```

Comment:
This would probably be better if we factored the two values out into functions, e.g.

```clojure
{:tables (get-table-metadata metadata)
 :quote  (get-identifier-quote metadata)}
```